core/drivers/crypto/caam/crypto.mk

```diff
@@ -75,6 +90,48 @@ $(call force,CFG_NB_JOBS_QUEUE, 80)  # Default JR index used
 $(call force,CFG_NXP_CAAM_SGT_V2,y)
 endif
 
+else # CFG_MX
```

Comment:
in fact CFG_MX does not exist, my comment is not correct and should probably be changed.

Reply:
maybe, but to prevent mixing configuration issue, it is then better to split in two dedicated config file :

ifeq ($(CFG_CAAM_LS),y)
include core/drivers/crypto/caam/crypto_ls.mk
endif
ifeq ($(CFG_CAAM_MX),y)
include core/drivers/crypto/caam/crypto_imx.mk
endif

?

Reply:
@etienne-lms  we already have CFG_NXP_CAAM and CFG_IMX_CAAM flags. I think that adding two more CAAM relative flags will introducing a bit more reading complexity in the config file.

Reply:
How about:
`# !CFG_LS, that is, MX family of platforms`